libs/vscode/webview/src/lib/webview.ts

```diff
@@ -178,6 +185,7 @@ export function getIframeHtml(
       window.vscode = acquireVsCodeApi();
     </script>
     <link href="${webView.asWebviewUri(stylePath)}" rel="stylesheet"/>
+    <link href="${webView.asWebviewUri(codiconsPath)}" rel="stylesheet" />
```

Comment:
I'm occasionally seeing a 404 on this codicon.css in the console, might just be showing up on dev mode rebuild but looking into it.

Reply:
Resolved by adding vscode-codeicons to vscode app package.json dependencies and referencing in the dist node_modules